### DISCUSSION TITLE
Angular project

### DISCUSSION THREAD

**Discussion by warszawa86** [community/community#79187] [Programming Help]
### Body

Why do I have so many problems of compability here?

{
  "name": "SALDE",
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "~8.0.1",
    "@angular/common": "~8.0.1",
    "@angular/compiler": "~8.0.1",
    "@angular/core": "~8.0.1",
    "@angular/fire": "^5.2.1",
    "@angular/forms": "~8.0.1",
    "@angular/platform-browser": "~8.0.1",
    "@angular/platform-browser-dynamic": "~8.0.1",
    "@angular/router": "~8.0.1",
    "@fortawesome/angular-fontawesome": "^0.5.0",
    "@fortawesome/fontawesome-svg-core": "^1.2.30",
    "@fortawesome/free-solid-svg-icons": "^5.14.0",
    "@ng-bootstrap/ng-bootstrap": "^5.0.0-rc.1",
    "angular-feather": "^6.0.2",
    "bootstrap": "^4.3.1",
    "chart.js": "^2.8.0",
    "core-js": "^3.1.4",
    "firebase": "^6.2.3",
    "jquery": "^3.4.1",
    "moment": "^2.27.0",
    "ng-bootstrap": "^1.6.3",
    "ng2-charts": "^2.3.0",
    "ng2-file-upload": "^1.4.0",
    "ngx-bootstrap": "^5.1.0",
    "ngx-dropzone": "^2.2.2",
    "ngx-mask": "^8.0.1",
    "ngx-ui-loader": "^8.0.0",
    "popper.js": "^1.15.0",
    "rxjs": "~6.5.2",
    "tslib": "^1.9.0",
    "zone.js": "~0.9.1"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.800.0",
    "@angular/cli": "~8.0.4",
    "@angular/compiler-cli": "~8.0.1",
    "@angular/language-service": "~8.0.1",
    "@types/node": "~8.9.4",
    "@types/jasmine": "~3.3.8",
    "@types/jasminewd2": "~2.0.3",
    "codelyzer": "^5.0.0",
    "jasmine-core": "~3.4.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.1.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.1",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "protractor": "~5.4.0",
    "ts-node": "~7.0.0",
    "tslint": "~5.15.0",
    "typescript": "~3.4.3"
  }
}

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by lc2101** [comment]
run 'npm install'. That always helps me with every change i do.

**Comment by abdullol** [comment]
Uninstalling your node module and re-installing it using ` npm install ` might solve your issue. Additionally, check the version of specific third-party libraries, like **ngx**, which might be causing issues.

**Comment by MuhamedHabib** [comment]
just npm uninstall then npm install --force
